pkg/annotations/annotations.go

```diff
@@ -31,6 +31,10 @@ const (
 
 	// OCISeccompBPFHookAnnotation is the annotation used by the OCI seccomp BPF hook for tracing container syscalls
 	OCISeccompBPFHookAnnotation = "io.containers.trace-syscall"
+
+	// MaybeSELinuxLabelAnnotation is the annotation used for optionally skipping relabeling a volume with the specified SELinux label.
+	// The relabeling will be skipped if the top layer is already labeled correctly.
+	MaybeSELinuxLabelAnnotation = "io.kubernetes.cri-o.MaybeSELinuxLabel"
```

Comment:
non blocking nit because this may already been discussed: I would prefer having something more specific here like "io.kubernetes.cri-o.SkipSELinuxLabel" or "io.kubernetes.cri-o.SkipVolumeSELinuxLabel"

Reply:
changed it to `TrySkipSELinuxLabel`

Reply:
does that work for you? also wondering @kolyshkin 's thought